component/loki/source/heroku/heroku.go

```diff
@@ -30,38 +30,13 @@ func init() {
 // Arguments holds values which are used to configure the loki.source.heroku
 // component.
 type Arguments struct {
-	HerokuListener       ListenerConfig      `river:"listener,block"`
```

Comment:
Some of these changes will require documentation updates too.

Reply:
Docs added in the follow up commits